### DISCUSSION TITLE
Lenght of the line

### DISCUSSION THREAD

**Discussion by ludvigtyden** [e2nIEE/pandapower#2245] [Q&A] [answered]
Hi, the function "pandapower.create_line(net, from_bus, to_bus, length_km, std_type, name=None, index=None, geodata=None, df=1.0, parallel=1, in_service=True, max_loading_percent=nan, alpha=nan, temperature_degree_celsius=nan, **kwargs)" takes length_km as input. Is that the length of the actual conductor or is it the lenght of the span between towers? or are they equivalent i.e the line has no sag?

**Comment by vogt31337** [comment, marked as answer]
@ludvigtyden , it is the length of the actual conductor. So if there are multple towers, it is the length of the whole line over all towers, including the sag.
